src/xdocs/index.xml.vm

```diff
@@ -97,8 +97,25 @@
         <a href="https://github.com/checkstyle/checkstyle/milestones">milestones</a>.
       </p>
       <p>
-        Checkstyle is not using Semantic Versioning
-        (<a href="https://github.com/checkstyle/checkstyle/issues/3709">reasons in issue #3709</a>)
+        Checkstyle is following
+        <a href="https://dafoster.net/articles/2015/03/14/semantic-versioning-vs-romantic-versioning/">
+          Hybrid Romantic and Semantic Versioning</a>:
+        <b>First</b>.<b>Second</b>.<b>Third</b>
+        <br/>
+        <b>First</b> First digit is representing Romantic version.
+           It means that some very noticable
+           (from maintainer's perspective) breaking compatibility happened
```

Comment:
`It means that some very noticable (from maintainer's perspective) breaking compatibility happened` => `When it is the only number increasing, the maintainers marked it as a noticeably large breaking compatibility or major conceptual change occurred from their perspective.`.